docs/guides/online-lab-environments.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Online lab environments
+description: Set up an online lab environment
+
+---
+<span id="online-environments"></span>
+# Online lab environments
+
+In addition to downloading and installing Qiskit to work on your local machine, there are several options available for those who prefer to work in an online development environment.
+
+Several providers offer online Jupyter environments with Qiskit pre-installed for fast and convenient onboarding. These recommended providers include the cloud service provider OVHcloud and the quantum software development cloud platform qBraid.
+
+Additional online Jupyter notebook environments like IBM Watson&reg; Studio, Google Colab, and Microsoft Azure Machine Learning Studio can be used to work with Qiskit by following the instructions below.
+
+## Recommended notebook environments with Qiskit pre-installed
+
+### OVH Cloud AI Notebooks
+
+Users of OVHcloud Public Cloud and OVHcloud Startup can now leverage their OVHcloud quantum notebooks to access Qiskit and submit jobs to IBM Quantum hardware with their IBM Quantum or IBM Cloud&reg; account.
+
+<Admonition type="note">
+OVHcloud AI notebooks are only available in select regions.
+</Admonition>
+
+Follow these steps to get started:
+
+1. Set up an OVHcloud account.
+
+    1. Visit the [OVH Cloud landing page.](https://us.ovhcloud.com/)
+    3. In the Public Cloud box, click **Learn more**.
+    4. In the **Public Cloud** section, click **Get Started**.
+    5. In the **I'm new to OVHcloud _xx_** box, enter your details and click **Create an account**.
+    7. After confirming your email address, finish creating the account.
+
+2. Create an OVH Public Cloud project.
+
+    1. Log in and navigate to the [Public Cloud page.](https://us.ovhcloud.com/manager/#/public-cloud/pci/projects/new?context=website)
+    2. Name your project and click **Create my project**.
+    3. ******************At this POINT YOU HAVE TO PAY.
```

Comment:
I assume you also have some clarity you want to provide here @beckykd, but just flagging it regardless (:

Reply:
Yes - good catch